game.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './item'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_date, :publish_date
+
+  def initialize(multiplayer, last_played_date, publish_date)
+    super(name, publish_date)
```

Comment:
- [ ] I think, you should remove the `name` parameter from the super() method; because we don't have `name` property in the Item class from which the Game class inherits.